admin/satellites/geo-dns.md

```diff
@@ -0,0 +1,120 @@
+---
+title: GeoDNS
+description: |
+    Learn how to configure your primary and satellite deployments to share a
+    single hostname.
+state: alpha
+---
+
+Out of the box, satellites deployments have a different access URL to the
+primary deployment. This may cause confusion for engineering teams located
+globally. This may cause confusion amongst users wondering which access URL they
+should access Coder from.
+
+Coder supports a GeoDNS configuration where all satellites and the primary
+deployment share the same hostname, so all users access Coder using a consistent
+URL but maintain low latency to their workspaces.
+
+> We recommend maintaining separate access URLs for each satellite and primary
+> deployment alongside the new GeoDNS access URL so you can access specific
+> deployments while avoiding GeoDNS in the future. This guide will walk you
+> through this configuration process.
+
+## Requirements
+
+You will need the following:
+
+- A primary access URL (e.g. `https://primary.example.com`).
+- One or more satellite access URLs (e.g. `https://sydney.example.com`,
+  `https://london.example.com`).
+- A GeoDNS load balancer access URL (e.g. `https://coder.example.com`). The
```

Comment:
GCP does something similar with Anycast, https://cloud.google.com/load-balancing

We should distinguish whether this feature is for people using Anycast (a CDN or global load balancing) versus split-horizon DNS (where the DNS responses vary according to the user's IP address/location)